cypress/e2e/requests.cy.js

```diff
@@ -22,21 +22,33 @@ describe('Viewing all requests', () => {
       // Intercept the response from the endpoint to view all requests
       // TODO(summer-cook): extract out this base url into the config to use as an environment variable. it was not cooperating before
       cy.intercept('GET', `${scientistApiBaseURL}/quote_groups/mine.json`, (req) => {
-        if ((requestList === undefined) && (loading === true)) {
+        switch (true) {
           // reply with an empty response: both data and error will be undefined.
-          req.reply()
-        } else if ((requestList === undefined) && (loading === false) && (error === true)) {
+          case loading : req.reply()
```

Comment:
does cypress require a space before the semi-colon on all of these case statements? if not, can you remove the space before, but leave the space after the semi-colon?

Reply:
no it doesnt! will do that before merging